test/uk/gov/hmrc/merchandiseinbaggage/content/CheckYourAnswerImportContentSpec.scala

```diff
@@ -22,12 +22,14 @@ import uk.gov.hmrc.merchandiseinbaggage.model.api.YesNo.{No, Yes}
 import uk.gov.hmrc.merchandiseinbaggage.model.api.calculation.CalculationResult
 import uk.gov.hmrc.merchandiseinbaggage.model.core.DeclarationJourney
 import uk.gov.hmrc.merchandiseinbaggage.smoketests.pages.CheckYourAnswersPage
-import uk.gov.hmrc.merchandiseinbaggage.stubs.MibBackendStub.givenAPaymentCalculation
+import uk.gov.hmrc.merchandiseinbaggage.stubs.MibBackendStub
 
 import scala.jdk.CollectionConverters._
 
 class CheckYourAnswerImportContentSpec extends CheckYourAnswersPage with CoreTestData {
 
+  private val mibStub = app.injector.instanceOf[MibBackendStub]
```

Comment:
Once the changes in the `MibBackendStub` have been done, please revert these changes, thanks